src/node_jsvmapi.h

```diff
@@ -116,80 +116,99 @@ enum napi_valuetype {
   napi_function,
 };
 
-// Environment
-NODE_EXTERN napi_env napi_get_current_env();
+enum napi_errorvalue {
```

Comment:
I can get behind that. napi_errorvalue is just a placeholder for a better name, anyway.